### PR TITLE
rustfmt on renderer crate

### PR DESCRIPTION
**Pull request by Rhuagh**

### CODE REVIEW

**Comment by Xaeroxe** [comment]
cc @omni-viral @torkleyy @ebkalderon

File: amethyst_renderer/src/config.rs [hunk 1]
```diff
@@ -45,10 +45,18 @@ impl Default for Config {
         }
     }
 }
-fn default_title() -> String { "Amethyst game".to_string() }
-fn default_vsync() -> bool { true }
-fn default_multisampling() -> u16 { 1 }
-fn default_visibility() -> bool { true }
+fn default_title() -> String {
```
Comment (Xaeroxe) [inline amethyst_renderer/src/config.rs]: Can we use `#[cfg_attr(rustfmt, rustfmt_skip)]` for these default functions?  I like the old formatting better.

Comment (Rhuagh) [reply amethyst_renderer/src/config.rs]: I'm fine with either.

Comment (omni-viral) [reply amethyst_renderer/src/config.rs]: I'd prefer to avoid skipping for consistency

Comment (Xaeroxe) [reply amethyst_renderer/src/config.rs]: I guess I can live with that.

Comment (Rhuagh) [reply amethyst_renderer/src/config.rs]: We can make it default to have single expression functions on a single line if you want.

Comment (Xaeroxe) [reply amethyst_renderer/src/config.rs]: That works!

Comment (Rhuagh) [reply amethyst_renderer/src/config.rs]: Can I fix that as a separate PR after all the current ones are merged? Getting messed up in the head from all the PRs..

Comment (Xaeroxe) [reply amethyst_renderer/src/config.rs]: Sure

**Pull request merged**